### ISSUE TITLE
Aliases for RFC 3339 formats are defined but not used

### ISSUE THREAD

**Issue by Stranger6667** [Zac-HD/hypothesis-jsonschema#79]
Hi!

In `rfc3339` I see the following conditions:

```python
if name == "date" or name == "full-date":
    return st.dates().map(str)
if name == "time" or name == "full-time":
    return st.tuples(rfc3339("partial-time"), rfc3339("time-offset")).map("".join)
```

The first comparisons are always false because of the following line at the beginning of the function:

```python
assert name in RFC3339_FORMATS
``` 

`date` and `time` are not in `RFC3339_FORMATS` + the `format` value is checked against `known_formats` in the `string_schema` function.

Found this when I was trying to generate data for Open API with `{"format": "date", "type": "string"}` in Schemathesis.

At the moment, I can extend Schemathesis with a custom format for `date`, but as these formats are a part of JSON Schema, I believe that those aliases should be added to `STRING_FORMATS`.

What do you think?

**Comment by Zac-HD** [comment]
Well spotted!  This will be a great first issue for someone at the Mentored Sprints this weekend 😄  The task: go to

https://github.com/Zac-HD/hypothesis-jsonschema/blob/d6b98395ccdd34963f6b9a80fc19697a7226dec2/src/hypothesis_jsonschema/_from_schema.py#L373-L375

and add a line `"date": rfc3339("full-date")` and the equivalent line for `"time"`.  That's it!

**Comment by Stranger6667** [comment]
Great! :) In the meanwhile, I'll temporarily add those formats to Schemathesis directly :)

**Comment by Zac-HD** [comment]
Fixed in #81, by @jjpal, and released as 0.19.2 🎉
